examples/decode.py

```diff
@@ -14,18 +14,25 @@ def main():
                         help='Output file (config.xml)')
     parser.add_argument('--key', type=lambda x: x.encode(), default=b'',
                         help="Key for AES decryption")
+    parser.add_argument('--serial', type=str, default='',
+                        help="Serial number for AES decryption (digimobil routers)")
     args = parser.parse_args()
 
-    key = args.key.ljust(16, b'\0')[:16]
+    if args.serial:
+        key = args.serial
```

Comment:
does it make sense to do the concatenation of the digimobil key to the serial number in the example? dunno what's cleanest - any thoughts?

Reply:
nah maybe not, as you need the IV as well... let's leave as-is :+1: